### PR TITLE
Tax identifier

### PR DESCRIPTION
**Pull request by jchen293**
Add tax identifier for Java Client Library

### CODE REVIEW

File: src/main/java/com/easypost/model/Shipment.java [hunk 1]
```diff
@@ -35,6 +35,7 @@
 	String uspsZone;
 	Map<String, Object> options;
 	List<ShipmentMessage> messages;
+    List<TaxIdentifier> taxIdentifiers;
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Shipment.java:38]: This is indented incorrectly, can we fix it? Also, it should be singular.
```suggestion
    List<TaxIdentifier> taxIdentifier;
```

File: src/main/java/com/easypost/model/Shipment.java [hunk 2]
```diff
@@ -225,6 +226,13 @@ public void setForms(List<Form> forms) {
 		this.forms = forms;
 	}
 
+	public List<TaxIdentifier> getTaxIdentifier(){
```
Comment (Justintime50) [inline src/main/java/com/easypost/model/Shipment.java:229]:
```suggestion
	public List<TaxIdentifier> getTaxIdentifiers(){
```

**Comment by Justintime50** [comment]
This still wasn't rebased correctly, it contains the already removed `.travis.yml` file removal which was already done.

This will need to be rebase properly.

**Comment by jchen293** [comment]
Created the taxIdentifier branch for PR.

**Pull request closed without merging**